cape/cape_result.py

```diff
@@ -111,7 +111,8 @@
     "WinHttpConnect",
     "WSASend",
 ]
-BUFFER_API_CALLS = ["send", "WSASend"]
+BUFFER_API_CALLS = ["send", "sendto", "recv", "recvfrom", "WSARecv", "WSARecvFrom", "WSASend", "WSASendTo", "WSASendMsg", "SslEncryptPacket", "SslDecryptPacket", "InternetReadFile", "InternetWriteFile"]
+CRYPT_BUFFER_CALLS = ["CryptDecrypt", "CryptEncrypt", "BCryptDecrypt", "BCryptEncrypt", "NCryptDecrypt", "NCryptEncrypt", "OutputDebugStringA", "OutputDebugStringW"]
```

Comment:
This should be split into two different constants since OutputDebugStringA and OutputDebugStringW are not CRYPT_BUFFER calls